1.10/metrics/quickstart.md

```diff
@@ -71,12 +71,12 @@ The metrics component is natively integrated with DC/OS and no additional setup
         The output should resemble this:
         
         ```json
-        ["121f82df-b0a0-424c-aa4b-81626fb2e369","87b10e5e-6d2e-499e-ae30-1692980e669a"]
+        ["28dfb041-42bb-4064-bd4f-bd56c472eb1e","d46f5b51-2adb-4978-bf8a-2a4a85103ab6"]
```

Comment:
It doesn't do any harm to change these, but they're sample values - they'll be different each time.

Reply:
I wanted them to match the examples below.